### ISSUE TITLE
[FEATURE]: Wapen verwijderen uit heupholster na vastpakken

### ISSUE THREAD

**Issue by Rick-TDA** [Tedeapolis/development#2535]
### Contact Details

rick.4524

### Goedgekeurd door

Korpsleiding

### Beschrijf zo duidelijk mogelijk de feature

In ons EUP hebben wij een heupholster. Zodra je je wapen uit je heupholster pakt is het wapen (de prop) nog te zien in de heupholster terwijl je deze roleplay wise vast hebt. 

Deze github heeft geen haast/prioriteit. Dit is enkel roleplay bevorderend.

![image](https://github.com/user-attachments/assets/d84665a5-e22d-407e-b962-3ea537ebea36)

### Wat lost deze feature op?

Realisme 

### Richtlijnen

- [X] Ik ga akkoord met de issue richtlijnen en heb deze aandachtig gelezen

**Comment by niekschoemaker** [comment]
Als je zelf even een nummer van de versie zonder wapen kunt opzoeken, als die uberhaupt bestaat.

Als er een nummer bij zit kan dit, als er geen versie is zonder het geweer erin kan dit helaas niet.

**Comment by Rick-TDA** [comment]
Yes, deze is er zeker. Het gaat om nummer **2** in de kledingwinkel onder het kopje **"ketting 1"**

![image](https://github.com/user-attachments/assets/d0b2e146-ca82-4535-8f71-4af1186c4ac6)

**Comment by Coolcooli** [comment]
fixed na merge
